src/helpers/certs-helpers.ts

```diff
@@ -0,0 +1,25 @@
+import { readdirSync, readFileSync, ensureDirSync } from 'fs-extra';
+import { join } from 'node:path';
+import { userDataCertsPath } from '../environment-remote';
+
+export function removeNewLines(string: string) {
+  return string.replaceAll(/\r?\n|\r/g, '');
+}
+
+export function readCerts() {
+  const certsFolder = userDataCertsPath();
+
+  ensureDirSync(certsFolder);
```

Comment:
What would happen here if no dir exists? Shouldn't we exit the flow then?

Reply:
Per the documentation, the dir is created if it doesn't exist (which is pretty cool, didn't know this existed before this)
https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir-sync.md

Reply:
Awesome, then no concerns here.